core/src/stored/sd_stats.cc

```diff
@@ -513,7 +513,7 @@ bool StatsCmd(JobControlRecord* jcr)
 
   if (job_statistics) {
```

Comment:
This ~~function~~ file does not make sense to me.

Why do we lock job_statistics when we remove something but not when we iterate over it ? This cannot be right.
Looking at it, this is also the case for the job_stats stuff.

This probably shouldnt get fixed in this pr though.